libs/deal-ticket/src/components/time-in-force-selector.tsx

```diff
@@ -44,7 +45,13 @@ export const TimeInForceSelector = ({
             timeInForce === VegaWalletOrderTimeInForce.FOK ||
             timeInForce === VegaWalletOrderTimeInForce.IOC
         );
-
+  useEffect(() => {
```

Comment:
TIF field is already selected on Market tab, so whenever order type updates we need to reselect TIF field.